ansible-sidecar/app/server.py

```
@@ -0,0 +1,181 @@
+        provided = header[7:] if header.startswith("Bearer ") else ""
+        return bool(TOKEN) and hmac.compare_digest(provided, TOKEN)
+
+    def _json(self, status: int, body: dict[str, Any]) -> None:
+        encoded = json.dumps(body).encode()
+        self.send_response(status)
+        self.send_header("Content-Type", "application/json")
+        self.send_header("Content-Length", str(len(encoded)))
+        self.end_headers()
+        self.wfile.write(encoded)
+
+    def _read_json(self) -> Any:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_read_json`

(ANN401)